### PR TITLE
Implement Onboarding, part 1

### PR DESCRIPTION
**Pull request by dherault**
## Summary

This is Klink's work on the onboarding flow.
I submit the PR now so that we keep all PRs small enough.

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Review by dherault — COMMENTED** [review]
Ahhh love it.

File: www/src/components/shell/cloud/InstallCli.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  return (
+    <>
+      <DemoCard title="Install Plural CLI">
+        <P
+          body1
+          color="text-light"
+          marginBottom="large"
+        >
+          Our <b>Quickstart (CLI)</b> documentation will bring you through the flow to get started on your local machine. After you’re done, move on to the next step to experience the rest of what Plural has to offer.
```
[the remark this answers is not in the document]
Comment (dherault) [reply www/src/components/shell/cloud/InstallCli.js:18]: strong
lol it htmled it

[the remark this answers is not in the document]
Comment (dherault) [reply www/src/components/shell/cloud/InstallCli.js:18]: > No closing tag, no problem - GitHub

File: www/src/components/shell/SplashToLogoTransition.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+export default function SplashToLogoTransition({ showSplashScreen = false, ...props }) {
+  if (showSplashScreen) {
+    return <LogoAndSplash {...props} />
+  }
+ 
+  return <LogoOnly>{props.children}</LogoOnly>
+}
+
+function LogoOnly({ children }) {
```
Comment (michaeljguarino) [inline www/src/components/shell/SplashToLogoTransition.js:102]: Why do we need children as part of this component?

File: www/src/components/Plural.js [hunk 3]
```diff
@@ -67,10 +86,6 @@ export function PluralInner() {
           <DeviceLoginNotif />
           <Routes>
             {/* --- OAUTH --- */}
-            <Route
```
[the remark this answers is not in the document]
Comment (dherault) [reply www/src/components/Plural.js:70]: It has been moved to a sub route, in another comp. Will take a look in another PR

[the remark this answers is not in the document]
Comment (dherault) [reply www/src/components/Plural.js:70]: Oh! look at line 65 it's there ( green part)

**Pull request merged**